applications/feedback_sentiment_analyzer/DEVELOPMENT.md

```diff
@@ -0,0 +1,76 @@
+# FSA development
+
+This document describes the process for adding an additional set of AWS Lambda functions for use with the Feedback Sentiment Analyzer (FSA). The requirements for each individual Lambda function are in [SPECIFICATION.md](./SPECIFICATION.md).
+
+## Add new Lambda function configurations
+
+The `FUNCTIONS` map in [functions.ts](./cdk/lib/functions.ts) maps an arbitrary deployment name to a list of Lambda functions required by FSA. This is the main entry point for defining your Lambda functions that will run with FSA. These functions are deployed when you run `cdk deploy` as described in the [README](./README.md). The following steps explain how to add new functions.
+
+1. Add a new entry to the `FUNCTIONS` constant. For example, adding JavaScript functions might look like this:
+
+```typescript
+const FUNCTIONS: Record<string, AppFunctionConfig[]> = {
+  examplelang: EXAMPLE_LANG_FUNCTIONS,
+  javascript: JAVASCRIPT_FUNCTIONS,
+};
+```
+
+2. `JAVASCRIPT_FUNCTIONS` is undefined. Define a constant named `JAVASCRIPT_FUNCTIONS` and set it equal to an empty `AppFunctionConfig` array.
+
+```typescript
+const JAVASCRIPT_FUNCTIONS: AppFunctionConfig[] = [];
+```
+
+3. Define the objects inside the `JAVASCRIPT_FUNCTIONS` array. Each object is an [AppFunctionConfig](./cdk/lib/constructs/app-lambdas.ts#4) which extends [FunctionProps](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda.FunctionProps.html) from the AWS Cloud Development Kit (AWS CDK) library. FSA requires four named Lambda functions:
+
+- ExtractText
+- AnalyzeSentiment
+- TranslateText
+- SynthesizeAudio
+
+A definition for the `ExtractText` function might look like the following:
+
+```typescript
+  {
+    ...BASE_APP_FUNCTION,
+    codeAsset() {
+      return Code.fromInline(`
+        exports.handler = async (event) => {
+          // implementation goes here
+        }
+    `);
+    },
+    name: "ExtractText",
+  },
+```
+
+The `codeAsset` function must return a [Code](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda.Code.html) object. You can include inline code or [create a bundled asset](#bundling).
+
+## Implement Lambda functions
+
+The following sections provide a general overview of the required Lambda functions. The accompanying CDK script orchestrates these functions as [AWS Step Functions](https://docs.aws.amazon.com/step-functions/latest/dg/welcome.html). For a detailed specification of these functions see [SPECIFICATION.md](./SPECIFICATION.md).
```

Comment:
Here you say SPECIFICATION.md but above you say "the specification". Let's pick one and keep it consistent.

Reply:
Fixed